src/coreComponents/python/modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py

```diff
@@ -0,0 +1,124 @@
+
+import meshio
+from meshio._mesh import CellBlock
+import numpy as np
+import argparse
+import logging
+
+
+def convert_abaqus_to_gmsh(input_mesh, output_mesh, verbose=False):
+    """
+    @brief Convert an abaqus mesh to gmsh 2 format, preserving nodeset information
+    @param input_mesh path of the input abaqus file
+    @param output_mesh path of the output gmsh file
+    @param verbosity flag to display optional logging information
+    """
+    # Open the logger
+    logging.basicConfig(level=logging.WARNING)
+    logger = logging.getLogger(__name__)
+    if verbose:
+      logger.setLevel(logging.INFO)
+
+    # Load the mesh
+    logger.info('Reading abaqus mesh...')
+    mesh = meshio.Mesh.read(input_mesh, "abaqus")
+
+    # Convert the element regions to tags
+    logger.info('Converting region tags...')
+    region_list = list(mesh.cell_sets.keys())
+    n_regions = len(region_list)
+    cell_ids = np.zeros(len(mesh.cells[0][1]), dtype=int) - 1
+    for ii, region in enumerate(region_list):
+        mesh.field_data[region] = [ii + 1, 3]
+        cell_ids[mesh.cell_sets[region][0]] = ii + 1
+
+    # Add to the meshio datastructure
+    mesh.cell_data['gmsh:physical'] = [cell_ids]
+    mesh.cell_data['gmsh:geometrical'] = [cell_ids]
+    if (-1 in cell_ids):
+        logger.warning('Some element regions did not convert correctly to tags!')
```

Comment:
Do we want to continue here? Or having a `-1` means that something is _wrong_?
Is it a `warning` or an `error`?

Reply:
I had previously done this with exceptions.  However, my preference would be to continue the conversion attempt and produce a file that can help in the debugging, with the -1 indicating that something went wrong.

I'm not sure if this error would ever show up since we are relying on meshio for the initial reading.  Any mesh file that is poorly formed enough to cause problems here, will probably cause problems in meshio.

Reply:
I've also updated the function docstring/warning messages to inform the user of this behavior

Reply:
> I'm not sure if this error would ever show up since we are relying on meshio for the initial reading. Any mesh file that is poorly formed enough to cause problems here, will probably cause problems in meshio.

Bugs happen, file systems can be joking ;)

I'm OK with that. Then I would advise you to make `convert_abaqus_to_gmsh` return some error code and `sys.exit( 
convert_abaqus_to_gmsh( ... ) )` on it.

That way an errored process would not pass silently.
We could put it in some automated process for ex.

Otherwise LGTM.